website/docs/getting-started/using-the-docker_host-environment-variable.md

```diff
@@ -0,0 +1,47 @@
+---
+id: using-the-docker_host-environment-variable
+title: Using the `DOCKER_HOST` environment variable
+description: Using the `DOCKER_HOST` environment variable can make it easier to migrate from Docker to Podman Desktop, as it allows you to continue using familiar Docker commands while taking advantage of the benefits of Podman.
+tags: [podman-desktop, getting-started]
+keywords: [podman desktop, podman, containers, docker_host, environment, variable]
+hide_table_of_contents: false
+---
+
+# Using the `DOCKER_HOST` environment variable
+
+Consider using the `DOCKER_HOST` environment variable to migrate transparently from Docker to Podman Desktop on all platforms.
+
+* Continue using familiar Docker commands.
+* Take advantage of the benefits of Podman.
+* Your tools, such as [Gradle](https://gradle.org/) or [Testcontainers](https://www.testcontainers.org/), communicate with Podman without reconfiguration.
+
+#### Prerequisites
+
+* Podman
```

Comment:
should we have link to the install guides?

Reply:
We need to have an aggregate landing page for the Podman installation first.